src/pages/[platform]/build-a-backend/functions/examples/create_user_profile_record/index.mdx

```diff
@@ -0,0 +1,157 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Create a user profile record',
+  description:
+    'Use an Auth Post Authentication trigger to automatically a user profile record',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+
+You can use `defineAuth` and `defineFunction` to create a [Cognito post confirmation Lambda trigger](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-post-confirmation.html) to create a profile record when a user is confirmed.
+
+<Callout info>
+
+A user is "confirmed" when they verify their account. Typically this happens when the user confirms their email via the verification email. The post confirmation handler will _not_ be triggered for federated sign-ins (i.e. social sign-in).
+
+</Callout>
+
+To get started, create a new directory and a resource file, `amplify/auth/post-confirmation/resource.ts`. Then, define the Function with `defineFunction`:
+
+```ts title="amplify/auth/post-confirmation/resource.ts"
+import { defineFunction } from '@aws-amplify/backend';
+
+export const postConfirmation = defineFunction({
+  entry: './post-confirmation-handler.ts',
```

Comment:
```suggestion
```

we can take this out. technically both `name` and `entry` are optional but we've been defaulting to include the name instead of no props